.pre-commit-config.yaml

```diff
@@ -3,6 +3,7 @@ repos:
     rev: v1.71.0
     hooks:
       - id: terraform_fmt
+      - id: terraform_wrapper_module_for_each
```

Comment:
@antonbabenko what does this due to users who do not use Terragrunt?

Reply:
Several things. It allows users to work with one module instead of individual, if they prefer, e.g.:
```
module "alb" {
  items = {
    one = "..."
    two = "..."
  }
  # ...
}
```

or use fewer duplicated files (`terragrunt.hcl`) if they use terragrunt or other similar tools like terragrunt where for some reasons `for_each` is not an option (I saw a custom/legacy wrapper recently).

I use this approach exclusively with terragrunt (S3 bucket and object modules) and it is a ~life-saver~ code-saver. Some people use it with our EC2 instance module, and I think it is ok to be used with any other module (if a user wants).

This is rather handy when managing lots of similar smaller resources - IAM users, SNS, SQS, Lambdas, etc.

Wrappers have no extra logic or conditions.